tabs/src/components/ui/component/Text.jsx

```diff
@@ -0,0 +1,332 @@
+        size="larger"
+      />
+    </div>
+  )
+}
+
+function TextAreaItem() {
+  return (
+    <div>
+      <Header as="h4" content="TextArea" />
+
+      <TextTag weight="regular" size="large">
+        A TextArea is a multi-line plan-text editing control.
+      </TextTag>
+      <Collapse>
+        <Code code={ TextAreaCode } />
+      </Collapse>
+      <TextArea 
+        inverted
+        resize="both" 
+        placeholder="No more than 10 characters can be here" 
+        maxLength={ 10 }
+      />
+    </div>
+  )
+}
+
+function ToolbarItem() {
+  const stateReducer = (prevState, action) => ({ ...prevState, [action]: !prevState[action] })
+  const [state, dispatch] = React.useReducer(stateReducer, {
+    bold: false,
+    italic: false,
+    more: false,
+    underline: false,
+  })
+
+  return (
+    <div>
+      <Header as="h4" content="Toolbar" />
+
+      <TextTag weight="regular" size="large">
+        A Toolbar is a container for grouping a set of controls, 
+        often action controls (e.g. buttons) or input controls (e.g. checkboxes).
+      </TextTag>
+      <Collapse>
+        <Code code={ ToolbarCode } />
+      </Collapse>
+
+      <Toolbar
+        aria-label="Default"
+        items={[
+          {
+            icon: (
+              <BoldIcon
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'bold',
+            kind: 'toggle',
+            active: state.bold,
+            title: 'Toggle bold',
+            onClick: () => dispatch('bold'),
+          },
+          {
+            icon: (
+              <ItalicIcon
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'italic',
+            kind: 'toggle',
+            active: state.italic,
+            title: 'Toggle italic',
+            onClick: () => dispatch('italic'),
+          },
+          {
+            icon: (
+              <UnderlineIcon
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'underline',
+            kind: 'toggle',
+            active: state.underline,
+            title: 'Toggle underline',
+            onClick: () => dispatch('underline'),
+          },
+          {
+            key: 'divider-1',
+            kind: 'divider',
+          },
+          {
+            icon: (
+              <FontSizeIcon
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'font-size',
+            title: 'Font size',
+          },
+          {
+            icon: (
+              <RemoveFormatIcon
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'remove-format',
+            title: 'Remove formatting',
+          },
+          {
+            key: 'divider-2',
```

Comment:
Did this key required?

Reply:
Thanks, this key is not required, I will remove it.